instance/static/html/instance/index.html

```diff
@@ -63,7 +68,13 @@
                                                ng-click="provision(selected.instance)">
               Reprovision
             </button>
-            <img src="/static/img/running.png" tooltip-placement="bottom" tooltip="Provisioning in progress..." alt="Provisioning in progress..." ng-switch-default />
+            <button class="button reprovision" ng-switch-when="provisioning_failure"
+                                               ng-click="provision(selected.instance)">
+              Reprovision
+            </button>
+            <i class="fa fa-2x fa-ellipsis-h" tooltip-placement="right" tooltip="Provisioning in progress" ng-switch-default></i>
+            <i class="fa fa-2x fa-times" tooltip-placement="right" tooltip="Provisioning failure" ng-switch-when="provisioning_failure"></i>
+            <i class="fa fa-2x fa-check" tooltip-placement="right" tooltip="Ready" ng-switch-when="ready"></i>
```

Comment:
@antoviaque  OK, just confirming before writing the code -- Do you mean that now we can rely on status_progress to display either the success or error icon, correct? But the code above would still be repeated in these two places, is that a problem?

Also, should the "Provisioning in progress" picture display all the time or only when the provisioning is in fact running? And what about the other statuses, like BOOTED, ACTIVE, etc.?